plugins/MosaicCamera/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+# This is the cmake config file for the MosaicCamera
+SET(MOSAICCAMERA_VERSION "0.0.0")
```

Comment:
Why version is 0.0.0?

Reply:
The simplest solution is 

`SET(MOSAICCAMERA_VERSION "${VERSION}")
`
It just grows with the program. (The separate version numbers comes from long time ago, when "dynamical plugins" were really external downloads.)